feat: Make autorestic configuration file path configurable

micxer opened the pull request:
This commit updates the autorestic configuration file path to use the `autorestic_config_file` variable instead of hardcoding `~/.autorestic.yml`. This change allows for greater flexibility and customization in specifying the location of the configuration file.

micxer committed:
feat: Make autorestic configuration file path configurable

This commit updates the autorestic configuration file path to use the `autorestic_config_file` variable instead of hardcoding `~/.autorestic.yml`. This change allows for greater flexibility and customization in specifying the location of the configuration file.

dbrennand commented:
Hey @micxer 👋🏻 

Thank you for your contribution to this role!

The changes look good to me. The lint and tests are failing, but looks completely unrelated to these changes. I'll take a look when I can.

micxer committed:
chore: Use latest versions of GitHub Actions

This commit updates the GitHub Actions workflow file to use version v24 of ansible/ansible-lint instead of the archived version v6 ansible/ansible-lint-action. Also it update the action/checkout to v4.

micxer commented:
Fixed it. Should be working now.

dbrennand committed:
docs: add new contributor

dbrennand reviewed (approved):
LGTM. Thank you @micxer for your contribution! 🙂
